benchmarks/ps-experiment/llama-8b/ps_k8s_stack.yaml

```diff
@@ -0,0 +1,303 @@
+---
+# Source: vllm-stack/templates/poddisruptionbudget.yaml
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: "vllm-pdb"
+  namespace: default
+spec:
+  maxUnavailable: 1
+---
+# Source: vllm-stack/templates/serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: "vllm-router-service-account"
+  namespace: default
+---
+# Source: vllm-stack/templates/secrets.yaml
+apiVersion: v1
```

Comment:
hf_token secret is not needed.

Reply:
The purpose here is to generate the YAML with minimal changes necessary to make it work for benchmarking. The focus is not on finding the most optimal settings. By keeping the changes minimal and targeted, it's easier to understand what was modified and to track differences between the original file and the one used in the benchmark. Therefore, I won't include unnecessary or unrelated changes in this YAML

similar to PVC settings etc

Reply:
Ok.